js/toDoController.js

```diff
@@ -0,0 +1,35 @@
+var toDoListApp = angular.module('toDoList', []);
+
+toDoListApp.controller('toDoController', function($scope, $window){
```

Comment:
Do not use implicit dependency injection as this will fail if your js file is minified.  Use the explicit array syntax as follows:
```
toDoListApp.controller('toDoController', ['$scope', '$window', function($scope, $window) {
  ...
}]);
```